data/data/gcp/network/firewall-control.tf

```diff
@@ -42,7 +42,7 @@ resource "google_compute_firewall" "master_ingress_https_from_health_checks" {
 
   allow {
     protocol = "tcp"
-    ports    = ["6443"]
+    ports    = ["6080", "22624"]
```

Comment:
rename the health check s/https/http/

Reply:
Sorry meant to say rename the firewall

Reply:
Maybe just drop the https altogether... so we can lump additional health_check ports here if necessary?

Reply:
nit: source ranges could be reduced since it's now external only.

Reply:
note that to support private we just restrict the firewall only to source IPS within the domain, so we aren't blocked from having a private LB.